### DISCUSSION TITLE
open meteo requests for mip installation?

### DISCUSSION THREAD

**Discussion by GavinDunse** [micropython/micropython#13391] [Ideas]
I have run it in python on windows to generate solar irradiance predictions for 16 days.
Was thinking about using this data to determine whether to switch on/off the electric hot water and let the solar tubes do the work.
Was wondering if it could be packaged for mip installation on esp8266ex for example

**Comment by jimmo** [comment]
@GavinDunse It should be fairly straightforward to access this using the `requests` library (already in `mip`, and installed by default on most network-capable boards).
